acceptance-tests/storage_test.go

```diff
@@ -42,4 +42,44 @@ var _ = Describe("Storage", Label("storage"), func() {
 
 		appOne.DELETE(blobName)
 	})
+
+	It("works with Spring and  GCP Cloud libraries", Label("spring"), func() {
+		By("creating a service instance")
+		bucketName := random.Name(random.WithPrefix("csb"))
+		serviceInstance := services.CreateInstance(
+			"csb-google-storage-bucket",
+			"default",
+			services.WithParameters(map[string]any{
+				"name": bucketName,
+			}),
+		)
+		defer serviceInstance.Delete()
+
+		By("pushing the unstarted app twice")
+		appOne := apps.Push(apps.WithApp(apps.JDBCTestApp), apps.WithTestAppManifest(apps.StorageTestAppManifest))
+		appTwo := apps.Push(apps.WithApp(apps.JDBCTestApp), apps.WithTestAppManifest(apps.StorageTestAppManifest))
+		defer apps.Delete(appOne, appTwo)
+
+		By("binding the apps to the storage service instance")
+		binding := serviceInstance.BindWithParams(appOne, `{"role":"storage.objectAdmin"}`)
```

Comment:
If we're releasing a new version anyway, we should consider a default role.

Reply:
We can talk about this in the standup. It's out of scope of this task, so if we agree to change it, I’ll do it in another PR